qa/tasks/ceph_manager.py

```diff
@@ -1329,6 +1329,26 @@ def tmp(x):
             except CommandFailedError:
                 self.log('Failed to get pg_num from pool %s, ignoring' % pool)
 
+    def run_cluster_cmd(self, args, stdout=None, stderr=None, stdin=None,
+                        check_status=True, wait=True, cwd=None, timeout=300):
```

Comment:
I think it makes sense to modify `raw_cluster_cmd` to call this new method? Also, please just do:

```suggestion
    def run_cluster_cmd(self, *args, **kwargs):
```

and then override only as required.

Reply:
I intended to get rid of `raw_cluster_cmd()` and `raw_cluster_cmd_result()` since the functionality provided by them now is only a subset of `run_cluster_cmd()` on a separate PR originally (separate because they are very frequently called in QA code).